packages/components/src/Typeahead/Typeahead.scss

```diff
@@ -176,6 +180,10 @@ $tc-typeahead-items-font-size: 1.5rem !default;
 			background: none;
 		}
 	}
+
+	&.selected {
+		background-color: rgba($scooter, 0.2);
```

Comment:
Warning rgba($scooter, 0.2) !== opacity 0.2 from the spec

Reply:
fixed, thanks